armi/nuclearDataIO/cccc/dlayxs.py

```diff
@@ -38,18 +38,20 @@ class DelayedNeutronData:
     """
     Container of information about delayed neutron precursors.
 
-    This info should be enough to perform point kinetics problems and to compute the delayed neutron fraction.
+    This info should be enough to perform point kinetics problems and to compute the delayed neutron
+    fraction.
 
-    This object represents data related to either one nuclide (as read from a data library)
-    or an average over many nuclides (as computed after a delayed-neutron fraction calculation).
+    This object represents data related to either one nuclide (as read from a data library) or an
+    average over many nuclides (as computed after a delayed-neutron fraction calculation).
 
-    For a problem with P precursor groups and G energy groups, delayed neutron precursor information includes:
+    For a problem with P precursor groups and G energy groups, delayed neutron precursor information
+    includes:
```

Comment:
this is a bit of a cliffhanger...

Reply:
I have updated it.

The docstring was rather curt/perfunctory.  What they were _trying_ to say was the "precursor information" was the three class attributes listed directly under that colon.